docs/tutorials/PickupGuide.md

```diff
@@ -0,0 +1,77 @@
+PickupGuide
+===========
+
+### From open.mp Wiki
+
+  
+
+
+### Define the pickupid
+
+The first thing to be done when creating pickups is creating a place to store their ID. This will be done in a global variable so it can be set when you create the pickup and read when you pick up a pickup, calling a callback with the ID of the pickup you picked up. For this example we will use the name "mypickup".
+
+new mypickup;
+
+
+### Creating the pickup
+
+There are two ways to create pickups. [CreatePickup](../functions/CreatePickup "CreatePickup") and [AddStaticPickup](../functions/AddStaticPickup "AddStaticPickup"). AddStaticPickup doesn't return an ID when it is created, can't be destroyed and can only be used under OnGameModeInit, so for this example we will use [CreatePickup](../functions/CreatePickup "CreatePickup").
+
+**The syntax for [CreatePickup](../functions/CreatePickup "CreatePickup") is:**
+
+**Parameters:**
+|model|The model you'd like to use for the pickup.|
+|--- |--- |
+|type|The pickup spawn type, see further down this page.|
+|Float:X|The X-coordinate for the pickup to show.|
+|Float:Y|The Y-coordinate for the pickup to show.|
+|Float:Z|The Z-coordinate for the pickup to show.|
+|Virtualworld|The virtual world ID of the pickup. A value of -1 will cause the pickup to display in all virtual worlds.|
+
+
+  
+For this example we will create a cash pickup at Grove Street.
+
+Now we need to decide on a model to appear in the world, there are lots of models to choose from, some are listed on the external site [here](https://dev.prineside.com/en/gtasa_samp_model_id "Objects"), here choose model number 1274 which is dollar sign.
+
+Finally we need a [Type](https://wiki.openmultiplayer.now.sh/docs/scripting/resources/pickuptypes "Types") for the pickup, on the same page with the pickup models is a list of pickup types describing what the various ones do. We want this pickup to disappear when you pick it up, so you can't pick it up repeatedly, but to reappear after a few minutes so you can pick it up again, type 2 does just this.
+
+Pickups are most commonly created when the script starts, in [OnGameModeInit](../functions/OnGameModeInit "OnGameModeInit") or [OnFilterScriptInit](../functions/OnFilterScriptInit "OnFilterScriptInit") depending on the script type, however it can go in any function (for example you could create a weapon drop script which would use OnPlayerDeath to create weapon pickups).
+
+So here is the code to create our pickup, and store the ID in 'mypickup':
+
+mypickup = [CreatePickup](../function/CreatePickup)(1274, 2, 2491.7900, \-1668.1653, 13.3438, \-1);
```

Comment:
```suggestion
\`\`\`
gMyPickup = CreatePickup(1274, 2, 2491.7900, \-1668.1653, 13.3438, \-1);
\`\`\`
```

Reply:
Ignore the backslashes, that's just an escape character because the comments are also written in Markdown haha